src/webView/index.ts

```diff
@@ -26,22 +31,76 @@ export class PlaygroundWebview {
     this.rebuildWebview();
   }
 
-  public updateHTML(html: string) {
+  public async updateLibraryDependencies(playgroundText: string) {
```

Comment:
Should we call this `updateManifest`? Since we'll add more things to this file over time.

Reply:
Renamed.